model.ipynb

```diff
@@ -75,6 +76,7 @@
    "metadata": {},
    "source": [
     "### Generative model\n",
+    "These calculations are a part of the structural-color package (see above)\n",
```

Comment:
Same here, not all generative model calculations are performed within the structcol package.